src/compiler/scala/tools/nsc/ast/Trees.scala

```diff
@@ -196,9 +196,9 @@ trait Trees extends scala.reflect.internal.Trees { self: Global =>
   override protected def xtransform(transformer: super.Transformer, tree: Tree): Tree = tree match {
     case DocDef(comment, definition) =>
       transformer.treeCopy.DocDef(tree, comment, transformer.transform(definition))
-    case SelectFromArray(qualifier, selector, erasure) =>
+    case SelectFromArray(qualifier, selector, erasure0) =>
       transformer.treeCopy.SelectFromArray(
-        tree, transformer.transform(qualifier), selector, erasure)
+        tree, transformer.transform(qualifier), selector, erasure0)
```

Comment:
because of equals semantics, hard to say they couldn't possibly intend erasure phase